packages/fluentui/react-northstar/src/components/SplitButton/SplitButton.tsx

```diff
@@ -185,30 +191,33 @@ class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>,
               on: [],
               open: this.state.open,
               trigger,
+              ...(this.props.alignMenuRight && { target: this.targetRef.current, position: 'below', align: 'end' }),
```

Comment:
```suggestion
              ...(this.props.alignMenuRight && { target: this.targetRef.current, align: 'end' }),
```
Probably it's enough for us to force only align